test/TalentLayer.ts

```diff
@@ -150,4 +150,50 @@ describe("TalentLayer", function () {
         const bobTid = await talentLayerID.walletOfOwner(bob.address)
         expect(jobRegistry.connect(dave).createJobFromEmployer(bobTid, 'cid')).to.be.revertedWith("You sould have a TalentLayerId")
     })
+
+    it("Alice the employer can create an Open job", async function(){
+
+        await jobRegistry.connect(alice).createOpenJobFromEmployer('cid')
+        const jobData = await jobRegistry.jobs(4)
+
+        expect(jobData.status.toString()).to.be.equal('4')
+        expect(jobData.employerId.toString()).to.be.equal('1')
+        expect(jobData.initiatorId.toString()).to.be.equal('1')
+        expect(jobData.employeeId.toString()).to.be.equal('0')
+        expect(jobData.jobDataUri).to.be.equal('cid')
+    })
+
+    it("Alice can assign an employee to a Open job", async function(){
+
+        await jobRegistry.connect(alice).createOpenJobFromEmployer('cid')
+        const bobTid = await talentLayerID.walletOfOwner(bob.address)
+        await jobRegistry.connect(alice).assignEmployeeToJob(5, bobTid)
+        const jobData = await jobRegistry.jobs(5)
+
+        expect(jobData.status.toString()).to.be.equal('0')
+        expect(jobData.employeeId.toString()).to.be.equal(bobTid)
+
+    })
+
+    it("Bob can confirm the Open job", async function(){
+
+        await jobRegistry.connect(alice).createOpenJobFromEmployer('cid')
+        const bobTid = await talentLayerID.walletOfOwner(bob.address)
+        await jobRegistry.connect(alice).assignEmployeeToJob(6, bobTid)
+        await jobRegistry.connect(bob).confirmJob(6)
+        const jobData = await jobRegistry.jobs(6)
+
+        expect(jobData.status.toString()).to.be.equal('1') 
+    })
+
+    it("Bob can reject an Open job", async function(){
+
+        await jobRegistry.connect(alice).createOpenJobFromEmployer('cid')
+        const bobTid = await talentLayerID.walletOfOwner(bob.address)
+        await jobRegistry.connect(alice).assignEmployeeToJob(7, bobTid)
+        await jobRegistry.connect(bob).rejectJob(7)
+        const jobData = await jobRegistry.jobs(7)
+
+        expect(jobData.status.toString()).to.be.equal('3') 
```

Comment:
at this time, we can test that alice can assign a new employee

Reply:
Ok test set up and ok